utils/cosmos.ts

```diff
@@ -246,7 +246,7 @@ export async function signSendNFT (
     address,
     targetAddress,
     classId,
-    [nftId],
+    nftIds,
     { memo }
   ) as DeliverTxResponse
   return res
```

Comment:
@williamchong seems we are going to have two `signSendNFTs`, where one sends multiple classId-nftId pairs with a one-to-one relationship for collection feature, and another for sending multiple nftIds with the same classId.
Do you have any suggestions on naming or reusing the function?

Reply:
How about keep repeating the same classId in the 1-to-1 version?